applications/DelaunayMeshingApplication/custom_processes/mesher_process.hpp

```diff
@@ -79,49 +79,49 @@ class MesherProcess : public Process
 
     /// this function is designed for being called at the beginning of the computations
     /// right after reading the model and the groups
-    void ExecuteInitialize() final
+    void ExecuteInitialize() override
     {
```

Comment:
These methods must be "final". Mesher processes can only call Execute(), other methods are not called. Therefore no override method is allowed.

Reply:
It was giving a warning suggesting to use override and not final. I will check it again

Reply:
![screenshot from 2018-09-13 17-15-00](https://user-images.githubusercontent.com/25818902/45497624-96471700-b778-11e8-8f97-d9572e81b928.png)

Reply:
I think it asks you to use both: `final override`

Reply:
Is this ok for you @josep-m-carbonell? with 'final override' the warnings do not appear...

Reply:
Yes sure, I didn't get this warning from the compiler, using"final" was enough..but "final override" I guess it means the same. Thank you @philbucher and @AFranci.